### PR TITLE
Update yt2mp3.is-an.app.json

### PR DESCRIPTION
**Pull request by avalynndev**
### What my project does ?
A  website that converts Youtube videos into mp3 files. Made using Next.js

### NOTE:
I'm sorry, I pasted the wrong domain in my vercel page. So pls accept these new changes (CNAME replaced by TXT)

### Link to Website
Link: https://youtube2mp3.vercel.app

### CODE REVIEW

File: domains/yt2mp3.is-an.app.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "email": "avalynndev@gmail.com"
   },
   "record": {
-    "CNAME": "cname.vercel-dns.com."
```
[the remark this answers is not in the document]
Comment (avalynndev) [reply domains/yt2mp3.is-an.app.json:11]: Wait, i removed that by mistake... Fixed it.

File: domains/yt2mp3.is-an.app.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
   },
   "record": {
     "CNAME": "cname.vercel-dns.com."
```
Comment (tarampampam) [inline domains/yt2mp3.is-an.app.json:11]:
```suggestion
    "CNAME": "cname.vercel-dns.com.",
```

Comment (avalynndev) [reply domains/yt2mp3.is-an.app.json]: I get this error:  ERROR: cannot have CNAME and TXT record with same name: yt2mp3.is-an.app
So, I need to remove the CNAME ?

**Comment by avalynndev** [comment]
I get this error: ERROR: cannot have CNAME and TXT record with same name: yt2mp3.is-an.app
So, I need to remove the CNAME ?

**Comment by tarampampam** [comment]
> I get this error: ERROR: cannot have CNAME and TXT record with same name: yt2mp3.is-an.app So, I need to remove the CNAME ?

Oh, dude, I forgot - try to create a separate file (named `verify.yt2mp3.is-an.app.json`) with the following content:

```json
{
  "$schema": "../schemas/domain.schema.json",
  "description": "A website that converts Youtube videos into mp3 files. Made using Next.js",
  "domain": "is-an.app",
  "subdomain": "yt2mp3",
  "owner": {
    "repo": "https://github.com/avalynndev/youtube2mp3",
    "email": "avalynndev@gmail.com"
  },
  "record": {
    "TXT": [ "vc-domain-verify=yt2mp3.is-an.app,9a50e70e577e932e63c6" ]
  },
  "proxy": false
}
```

And keep the existing file with the `cname` record.

**Comment by avalynndev** [comment]
Thanks Bro...

**Comment by avalynndev** [comment]
But, After creating the new `verify` file, I Get this (Github Actions): 
```
ERROR: cannot have CNAME and TXT record with same name: yt2mp3.is-an.app
```

**Comment by tarampampam** [comment]
Yea, I see. Give me some time to fix it, please

**Comment by avalynndev** [comment]
Sure👍

**Comment by avalynndev** [comment]
Are you Quite done yet ?

**Comment by tarampampam** [comment]
Currently, I'm busy with my offline job. Can you wait some (3..4) hours?

**Comment by avalynndev** [comment]
Sure, please try to finish it by 2morrow.

**Comment by tarampampam** [comment]
@avalynndev could you provide full instructions for your domain verification?

According to [RFC 1034](https://www.rfc-editor.org/rfc/rfc1034):

> If a CNAME RR is present at a node, no other data should be
> present; this ensures that the data for a canonical name and its aliases
> cannot be different.

**Comment by tarampampam** [comment]
ping @avalynndev

**Comment by avalynndev** [comment]
Hi @tarampampam,
So, After a Bit of Research (I Just Asked some Friends). I Don't Think I need the verify and stuff. I need the TXT record only. So I am Doing the following:
- Deleting the `verify.ytmp3.is-an.app.json`
- Changing the CNAME Record to TXT in the `ytmp3.is-an-app.json` file

**Comment by tarampampam** [comment]
As you wish! Could you modify the [existing file](https://github.com/tarampampam/free-domains/blob/da36e6084b64267ea3c41f2d6932994685f4dd19/domains/yt2mp3.is-an.app.json#L1-L14) instead of creating a new one?

**Comment by tarampampam** [comment]
Anyway, I can't understand how it will work without A, AAAA, or CNAME records.

**Comment by avalynndev** [comment]
Same I don't understand how it is going to work, But that is what Vercel Says..

**Comment by tarampampam** [comment]
Just show the requirements for the domain verification, please. Kinda screenshot or something like that

**Comment by avalynndev** [comment]
Sure Bro

**Comment by avalynndev** [comment]
image

**Comment by tarampampam** [comment]
Holy shit, so trivial 😄 Press the "Refresh" button now

**Comment by avalynndev** [comment]
I did... at is works ????

**Comment by avalynndev** [comment]
Uhh.. Closing the Pr ?

**Comment by tarampampam** [comment]
All works fine now?

**Pull request closed without merging**